modules/afmongodb/afmongodb-private.h

```diff
@@ -44,6 +44,10 @@ typedef struct _MongoDBDestDriver
   const gchar *const_db;
   mongoc_uri_t *uri_obj;
   mongoc_client_pool_t *pool;
+
+  gboolean mongoc_opt_use_bulk;
+  gboolean mongoc_opt_unordered;
+  gboolean mongoc_opt_unacknowledged;
```

Comment:
This struct is all about Mongo, please consider removing the `mongoc_opt` prefix.
Using the same name you introduced in the grammar would be the best in my opinion, for example:

`use_bulk, bulk_unordered, write_concern`